tests/support/setup.tex

```diff
@@ -1,4 +1,8 @@
 \long\def\TYPE#1{%
+  \catcode"09=12%  Prevent tabs (U+0009) from folding into a single space token
+  \catcode"20=12%  Prevent spaces (U+0020) from folding into a single space token
+  \catcode"7C=12%  Prevent pipes (U+007C) from being active in ConTeXt
+  \catcode"7E=12%  Prevent tildes (U+007E) from being active in programmatic text
```

Comment:
Perhaps I wasn't clear in https://github.com/lostenderman/markdown/pull/137#issuecomment-1472745883, but this is the opposite of what I proposed, i.e. the catcode changing should go into the `documentBegin` and `documentEnd` renderers in `plain-setup.tex` and `markdownthemewitiko_markdown_test.sty`. This should not work, because by the time the redefinition takes place, `#1` has already been tokenized with and the tokens assigned category codes. The catcode changing should happen at the very beginning of a document.